src/databaseInterface/src/main/kotlin/org/vaccineimpact/api/db/LocalDateTimeConverter.kt

```diff
@@ -0,0 +1,30 @@
+package org.vaccineimpact.api.db
+
+import org.jooq.Converter
+import java.time.Instant
+import java.time.LocalDateTime
+import java.time.ZoneId
+import java.time.ZoneOffset
+
+class InstantConverter : Converter<LocalDateTime, Instant>
```

Comment:
This is a new class. The latest Jooq maps our Postgres timestamps to `LocalDateTime` and all our Kotlin classes want things of type `Instant` so I wrote this custom converter so that the conversion just happens automatically on generation.